nftproject/contracts/NFT.vy

```diff
@@ -8,23 +8,25 @@ implements: ERC721
 
 ############ ERC-165 #############
 # @dev Static list of supported ERC165 interface ids
-SUPPORTED_INTERFACES: constant(bytes4[4]) = [
-    0x01ffc9a7,  # ERC165 interface ID of ERC165
-    0x80ac58cd,  # ERC165 interface ID of ERC721
-    0x5b5e139f,  # ERC165 interface ID of ERC721 Metadata Extension
-    0x5604e225,  # ERC165 interface ID of ERC4494
+SUPPORTED_INTERFACES: constant(bytes4[6]) = [
+    0x01ffc9a7,  # interface ID of ERC165
+    0x80ac58cd,  # interface ID of ERC721
+    0x5b5e139f,  # interface ID of ERC721 Metadata Extension
+    0x5604e225,  # interface ID of ERC 4494
+    0x150b7a02,  # interface ID of ERC721 Token Receiever
+    0x780e9d63,  # interface ID if ERC721 Enuermable
 ]
 
 ############ ERC-721 #############
 
 # Interface for the contract called by safeTransferFrom()
 interface ERC721Receiver:
     def onERC721Received(
-            operator: address,
-            owner: address,
-            tokenId: uint256,
-            data: Bytes[1024]
-        ) -> bytes32: view
+        _operator: address,
+        _owner: address,
+        _tokenId: uint256,
+        _data: Bytes[1024]
+    ) -> bytes32: view
```

Comment:
I think technically this returns bytes4 per the standard

Reply:
willl change for 0.3.4